XCUITests/NavigationTest.swift

```diff
@@ -345,6 +345,84 @@ class NavigationTest: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
     }
 
+	func testLongPressOnAddressBar() {
+		//Navigate to a URL and perform a long press on the non-highlighted address bar
+		navigator.goto(URLBarOpen)
+		app.textFields["address"].typeText("www.google.com\n")
+        waitUntilPageLoad()
+		app.textFields["url"].tap()
+		app.textFields["address"].press(forDuration: 2)
+
+		//Ensure that long press on address bar brings up a menu with Select All, Select, Paste, and Paste & Go
+        waitForExistence(app.menuItems["Select All"], timeout: 3)
+		XCTAssertTrue(app.menuItems["Select All"].exists)
+        XCTAssertTrue(app.menuItems["Select"].exists)
+        if UIPasteboard.general.string != nil {
```

Comment:
If I'm not wrong, the clipboard is empty when starting the test, so what is inside this if is never going to be checked. 
My approach to the test would be a little bit simpler to save some steps:
- Long press and check the options (for iPhone and iPad)
- Long press copy text and check the options (Now Paste and Paste&Go should be shown)(for iPhone and iPad)

Reply:
The clipboard isn't always empty. It depends what tests would run before. The logic for what's in the pasteboard is dependent on how iOS handles its own pasteboard- we don't control it. So if a test before copy/pasted on the same sim- or on the same device- without a full system reset to clear the keyboard- we will encounter things in the keyboard. And there are two different UI flows for keyboard empty or not.